lib/tasks/migration.rake

```diff
@@ -702,9 +714,38 @@ namespace :migration do
       puts "Summary: Save file time: #{save_time}"
       puts "Summary: Add to Collection time: #{collection_time}"
       puts "Summary: Verification time: #{verify_time}"
-
+      noid_report.close
   end
 
+    def update_handle(handle,noid)
+      link = "https://era.library.ualberta.ca/files/#{noid}"
+      handle.slice!("http://hdl.handle.net/")
+      MigrationLogger.info "Deleting existing handle #{handle}"
+      puts handle
+      delete_cmd = "./bin/handle/bin/hdl-delete 0.NA/10402 bin/handle/lib/admpriv.bin #{handle}"
```

Comment:
How will the handle cli be installed in this path? Can we guarantee it will be? I don't see it in the project's bin directory currently or as part of this PR.